apps/sms/views/inbox/js/inbox.js

```diff
@@ -140,6 +147,10 @@ var InboxView = {
     if (args.notifyAboutSavedDraft) {
       this.showDraftSavedBanner();
     }
+
+    if (!args.meta.prev) {
+      this.renderThreads();
+    }
```

Comment:
wondering if we can not fix it here instead?

Some ideas:
* force that renderThreads runs only once -- `if (!renderingOrRendered) { this.renderThreads() }`
* do not call it here at all (for now) -- use apps/sms/views/conversation/js/startup.js instead

Both these ideas seem better than your current fix.

What do you think ?